test/functional/deprecated_rpc.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test deprecation of RPC calls."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+class DeprecatedRpcTest(BitcoinTestFramework):
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 2
+        self.setup_clean_chain = True
+
+    def setup_network(self):
```

Comment:
You don't need to override this method if you set `self.extra_args` in the `__init__()` method. Add the following line to the end of `_init__()` and remove the `setup_network()` method override:

```python
self.extra_args = [[], ["-enablerpcmethod=estimatefee"]]
```